web-api/src/Kerbero.Identity/Modules/Authentication/Services/AuthenticationService.cs

```diff
@@ -26,11 +26,11 @@ IAuthenticationManager authenticationManager
   public async Task<AuthenticatedModel> Login(LoginDto loginDto)
   {
     var user = await _userManager.FindByEmail(loginDto.Email);
-    if (user is null)
+    if (user is null || !user.EmailConfirmed)
```

Comment:
```suggestion
    if (user is null)
```
"options.SignIn.RequireConfirmedEmail = true;" should be enough